index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="fr">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Projets</title>
+</head>
+
+<body>
+    <div class="office ">
+        <p><strong>Clétus Officiel</strong></p>
+    </div>
+
+    <nav>
+        <ul>
+            <li>
+                <a href="#">Accueil</a>
+            </li>
+            <li>
+                <a href="#">Projets</a>
+            </li>
+            <li>
+                <a href="#">Album</a>
+            </li>
+            <li>
+                <a href="#">Contact</a>
+            </li>
+        </ul>
+    </nav>
+
+    <hr>
+
+    <div class="div">
+        <h3>Nos dernières actualités</h3>
+
+        <div class="first-projet ">
+            <div class="second">
+                <img src="images/th.jpg" alt="" class="image">
+            </div>
+            <div class="text">
+                <p>
+                    <span>Lorem ipsum</span><br>
+                    Dolor sit amet consectetur adipisicing <br> elit. Libero corrupti voluptatem, a in <br>alias
+                    debitis.Libero corrupti voluptatem.
+                </p>
+                <p class="last">
+                    Last udated 3 mind ago
+                    <a href="#" class="learn">Voir plus</a>
+                </p>
```

Comment:
il faut séparere le texte "Update..." du "Voir plus". tu peux faire quelque chose comme ci-dessous:
```suggestion
                <div>
                <p>Last udated 3 mind ago</p>
                 <a href="#" class="learn">Voir plus</a>
                </div>
```